src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -78,7 +80,9 @@
 
         // BankID
 
+        var cancelled;
```

Comment:
Propse chaning this to: `loginIsCancelledByUser` to make it more clear what kind of cancel it is :)

Reply:
Ok!